### PR TITLE
requirements file

### PR DESCRIPTION
**Pull request by ritvikrao**
Addresses issue #255, adding a requirements file to make the environment setup easier.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+cython>=3.0.0
+numpy>=1.10.0
+greenlet>=3.0.0
+cmake
```
Comment (matthiasdiener) [inline requirements.txt:4]: Thanks! These should probably match `install_requires` in setup.py?

Comment (matthiasdiener) [inline requirements.txt:4]: Maybe add a comment to the cmake line that cmake is needed for the Charm++ build.

Comment (ritvikrao) [reply requirements.txt]: done

Comment (ritvikrao) [reply requirements.txt]: done

**Review by matthiasdiener — APPROVED** [review]

**Pull request merged**